sdks/python/apache_beam/utils/dependency.py

```diff
@@ -58,15 +58,14 @@
 import shutil
 import tempfile
 
-
+from apache_beam.version import __version__
```

Comment:
It looks like this import was superfluous.  This is the one being used:

`from apache_beam import version as beam_version`

Can we remove `from apache_beam.version import __version__`?